input/pagecontent/specifications_techniques_14.md

```diff
@@ -0,0 +1,379 @@
+-	L’identifiant fonctionnel de la ressouce créée.
+
+
+##### Réponse de base -- Echec
+
+Lien vers la spécification FHIR :
+[https://www.hl7.org/fhir/R4/operationoutcome.html](https://www.hl7.org/fhir/R4/operationoutcome.html)
+
+Si la création échoue, le serveur doit répondre :
+
+Un header avec un un code erreur HTTP 4XX ou 5XX
+
+Un body contenant une ressource OperationOutcome qui donne les
+détails sur la raison de l'échec
+
+Remarque : l'échec d'une recherche est la non-possibilité d'exécuter la
+requête, ce qui est différent d'aucune correspondance à la recherche.
+Plus de précision sur la spécification FHIR :
+[https://www.hl7.org/fhir/R4/http.html](https://www.hl7.org/fhir/R4/http.html)
+
+#### Search
+
+##### Réponse de base -- Succès
+
+Lien vers la spécification FHIR : <http://hl7.org/fhir/R4/bundle.html>
+
+Si la recherche est un succès, le serveur répond :
+
+-   Un header avec un code 200 OK HTTP
+
+-   Un body contenant une ressource Bundle[^27] dont le type = searchset.\ Le bundle encapsule 0 à n ressources Measure corespondant aux critères de recherche plus les ressources incluses correspondant aux critères de recherche.\ Le service développé renvoie les 200 premiers résultats et indique le total trouvé dans une balise \"total\". Dans le cas où il n'y a pas de résultat le service renvoie \"total\": 0.
+
+Remarque : la recherche est un succès à partir du moment où la requête peut être exécutée. Il peut il y avoir 0 à n correspondances.
+
+Plus de précision sur la spécification FHIR  <http://hl7.org/fhir/R4/http.html>
+
+##### Réponse de base -- Echec
+
+Lien vers la spécification FHIR : <http://hl7.org/fhir/R4/operationoutcome.html>
+
+Si la recherche échoue, le serveur doit répondre :
+
+-   Un header avec un un code erreur HTTP 4XX ou 5XX
+
+-   Un body contenant une ressource [OperationOutcome](http://hl7.org/fhir/R4/operationoutcome.html) qui donne les détails sur la raison de l'échec
+
+Remarque : l'échec d'une recherche est la non-possibilité d'exécuter la requête, ce qui est différent d'aucune correspondance à la recherche. Plus de précision sur la spécification FHIR : <http://hl7.org/fhir/R4/http.html>
+
+### Critères de recherche pour Search
+
+-   Les critères de recherche, définis dans la spécifications FHIR [search parameters](https://hl7.org/fhir/R4/questionnaire.html#search) doivent être implémentés afin de pouvoir rechercher les fiches de saisie selon le besoin.
+
+### Paramètres et modificateurs de requêtes FHIR
+
+Les paramètres et modificateurs de requêtes sont décrits [ici](modifiers.html).
+
+### Exemple de requêtes
+
+#### Scénario 1 : Création d'une fiche de consignes de saisie
+
+**Description du scénario :** Un consommateur souhaite créer une nouvelle fiche de consignes de saisie pour un cabinet de ville de médecine générale.
+
+Exemple de ressource : [Example Questionnaire: Cabinet de ville de médecine générale](Questionnaire-ror-questionnaire-offre-de-ville.html)
+
+**Requête :**
+
+```json
```

Comment:
Ca va être très complexe à maintenir : si tu dois modifier la ressource questionnaire, tu dois aussi la modifier ici ?
Je pense que le lien suffit non ?

Au pire des cas : peut être que tu peux l'obtenir avec une requête sql, je peux t'aider à la construire si besoin!

Reply:
> Ca va être très complexe à maintenir : si tu dois modifier la ressource questionnaire, tu dois aussi la modifier ici ? 
oui mais demande du ROR sur les POST
> Je pense que le lien suffit non ?
A mon sens oui
> Au pire des cas : peut être que tu peux l'obtenir avec une requête sql, je peux t'aider à la construire si besoin!
why not

Reply:
Tu peux tenter (en remplaçant ExampleMesFrObservationBP001 par l'id de la ressource d'intérêt) :


{% sql SELECT Web FROM Resources WHERE Id = "ExampleMesFrObservationBP001" %}

Reply:
pas lisible :(